src/configure/mod.rs

```diff
@@ -47,17 +47,19 @@ pub fn dc_is_configured(context: &Context) -> bool {
 // the other dc_job_do_DC_JOB_*() functions are declared static in the c-file
 #[allow(non_snake_case, unused_must_use)]
 pub unsafe fn dc_job_do_DC_JOB_CONFIGURE_IMAP(context: &Context) {
+    if !context.alloc_ongoing() {
+        progress!(context, 0);
```

Comment:
i think, a `return` or so is missing here

Reply:
added.